front/src/modules/ui/field/meta-types/input/components/__stories__/FieldInputContextProvider.tsx

```diff
@@ -0,0 +1,35 @@
+import {
```

Comment:
is this different from FieldDisplayContextProvider? Looks very similar

Reply:
Fixed it